packages/nimble-tokens/source/styledictionary/config.js

```diff
@@ -228,128 +228,115 @@ const getTokenNameFromProperty = function (prop) {
-StyleDictionary.registerTransformGroup({
-    name: 'css',
-    transforms: [
-        'attribute/cti',
-        'name/dsp/kebab', // replaces 'name/cti/kebab',
-        'time/seconds',
-        'content/icon',
-        'size/rem',
-        'color/css'
-    ]
+StyleDictionary.registerTransform({
+    type: 'value',
+    transitive: true,
+    name: 'font/weight',
+    filter: token => token.attributes.category === 'font',
+    transform: token => {
+        if (token.value === 'Light') {
+            token.value = '300';
+        } else if (token.value === 'Regular') {
+            token.value = '400';
+        } else if (token.value === 'Semibold') {
+            token.value = '600';
+        } else if (token.value === 'Bold') {
+            token.value = '700';
+        }
+        return token.value;
+    }
 });
 
+// Combination of DSP & Nimble transform overrides
 StyleDictionary.registerTransformGroup({
-    name: 'scss',
+    name: 'nimble/css',
     transforms: [
         'attribute/cti',
-        'name/dsp/kebab', // replaces 'name/cti/kebab',
+        'name/nimble/kebab', // replaces name/dsp/kebab from DSP config
         'time/seconds',
-        'content/icon',
-        'size/rem',
+        'html/icon',
+        'size/px', // replaces size/rem from DSP config
         'color/css',
+        'font/weight'
     ]
 });
 
+// Combination of DSP & Nimble transform overrides
 StyleDictionary.registerTransformGroup({
-    name: 'android',
+    name: 'nimble/scss',
     transforms: [
         'attribute/cti',
-        'name/dsp/snake', // replaces 'name/cti/snake',
-        'color/hex8android',
-        'size/remToSp',
-        'size/remToDp',
-    ]
-});
-
-StyleDictionary.registerTransformGroup({
-    name: 'ios',
-    transforms: [
-        'attribute/cti',
-        'name/dsp/pascal', // replaces 'name/cti/pascal',
-        'color/UIColor',
-        'content/objC/literal',
-        'asset/objC/literal',
-        'size/remToPt',
-        'font/objC/literal',
-    ]
-});
-
-StyleDictionary.registerTransformGroup({
-    name: 'ios-swift',
-    transforms: [
-        'attribute/cti',
-        'name/dsp/camel', // replaces 'name/cti/camel',
-        'color/UIColorSwift',
-        'content/swift/literal',
-        'asset/swift/literal',
-        'size/swift/remToCGFloat',
-        'font/swift/literal',
+        'name/nimble/kebab', // replaces name/dsp/kebab from DSP config
+        'time/seconds',
+        'html/icon',
+        'size/px', // replaces size/rem from DSP config
+        'color/css',
+        'font/weight'
     ]
 });
 
+// Combination of DSP & Nimble transform overrides
 StyleDictionary.registerTransformGroup({
-    name: 'js',
+    name: 'nimble/js',
     transforms: [
         'attribute/cti',
         'name/dsp/pascal', // replaces 'name/cti/pascal',
-        'size/rem',
+        'size/px', // replaces size/rem from DSP config
         'color/hex',
-    ]
-});
-
-StyleDictionary.registerTransformGroup({
```

Comment:
removed the transform groups previously generated by adobe xd dsp plugin that we aren't using